t/831_xparse.t

```diff
@@ -0,0 +1,10 @@
+# -*- CPERL -*-
+#**********************************************************************
+# Test cases for LaTeXML
+#**********************************************************************
+use LaTeXML::Util::Test;
+use LaTeXML::Util::Pathname;
+
+latexml_tests("t/xparse",
+   texlive_min => 2018,
+   requires => {xparse => ['expl3.sty','xparse.sty'] });
```

Comment:
But, but... I thought I was joking. Or maybe I wasn't? Could actually crop up in the future.
However, it's kind of a shame that it is at the test-set level, rather than at the test case level (ie. it requires a separate directory of tests).That said, a natural evolution of the requires spec isn't immediately obvious.  Maybe something like:
```
requires => {xparse => {packages=>['expl3.sty','xparse.sty'], texlive=>2015} }
```
?

Reply:
Certainly doable. Should I refactor to that state?

Reply:
That said, why not have the option on both levels? The way I added you allows you to quickly skip an entire test set. Your suggestion is very granular to individual test files, but if you have say 10 different `.tex` tests for xparse, that means writing the `texlive=>` option ten times, once per file.

It's also the case that I really need `texlive_min` since both 2018 and 2019 are allowed at the moment. So maybe mergeable as-is ? It can be extended later.

Reply:
Sure, both levels works for me. But I'd still like to refactor rather than split off a new test directory (already too many). Actually, your top-level comment makes me think one might want to require a package for a whole directory (eg. babel), but the current terminology hijacks "requires".  Also, given the usage pattern, writing test sets/cases should be Easy; while Test.pm can be complex as it's seldom touched (and incomprehensible anyway)

Reply:
Alright, tests pass, left the global switch, but the xparse test now should look exactly as you want it:

```perl
latexml_tests("t/expl3",
   requires => {
      tilde_tricks => 'expl3.sty',
      xparse => {
         texlive_min => 2018,
         packages => ['expl3.sty','xparse.sty']
      } } );
```

Reply:
Beautiful! Thanks for accommodating :>